src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -272,6 +273,71 @@ private static void AddAntiForgery(OrchardCoreBuilder builder)
             });
         }
 
+        /// <summary>
+        /// Adds backwards compatibility to the handling of SameSite cookies.
+        /// </summary>
+        public static void AddSameSiteCookieBackwardsCompatibility(OrchardCoreBuilder builder)
```

Comment:
Is it worth exposing this method as a public API? I'm tempted to say we should keep it private.

Reply:
In our senario, we need it to be public. Since we use a different flavor of services. Making it private would result in us having to redo this feature.

Reply:
You mean you don't even use `OrchardCoreBuilder AddOrchardCore(this IServiceCollection services)`? `AddSameSiteCookieBackwardsCompatibility()` takes an `OrchardCoreBuilder` instance, that you're not really supposed to build manually. If you really want to use your own stuff, then you have to register the services `AddOrchardCore` normally registers for you, same-site stuff included.

All the other methods in this class are private, I don't see any compelling reason to make this one public, personally.

Reply:
Ok, will do.